### DISCUSSION TITLE
Disconnect but keep result in query

### DISCUSSION THREAD

**Discussion by laurent512** [prisma/prisma#13904] [Q&A]
Hello, 

Is there  away to keep information in query where fields have been disconnected.

For example here : 
```
 const query_result = await prisma.transactionValidationKey.update({
        where: { id: transactionValidationKey },
        data: {
          voucher: {
            update: {
              balanceEur: { decrement: amountEur },
              transactionsIssued: {
                create: {
                  amountEur: amountEur,
                  toId: user.shopId,
                },
              },
            },
            disconnect: true,
          },
        },
      });
```
This query does not return any information about the voucher, because it has been disconnected after the nested update.
How can I include it ?

**Comment by laurent512** [comment]
The solution I could consider would be to disconnect in a second query, but let's say that no ideal I think... 
And also I would like to be sure that the whole query worked before disconnecting

**Reply by SirJosh1987** [reply]
> The solution I could consider would be to disconnect in a second query, but let's say that no ideal I think... 
> 
> And also I would like to be sure that the whole query worked before disconnecting 
> 
> 
> 
> 

How would I do that?

**Comment by austincrim** [comment]
Hey @laurent512 👋,

Have you tried using the [`include` option](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#include) in your query?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
